docs/ManualDeploymentGuideScalarDLonAWS.md

```diff
@@ -0,0 +1,232 @@
+    
+   ```console
+    $ kubectl create secret docker-registry reg-docker-secrets --docker-server=ghcr.io --docker-username=<github-username> --docker-password=<github-personal-access-token>
+   ```
+
+3. Run the helm commands on the host machine to install Scalar DL on EKS
+    
+   ```console
+    # Add helm charts 
+    $ helm repo add scalar-labs https://scalar-labs.github.io/helm-charts
+    
+    # List the Scalar charts.
+    $ helm search repo scalar-labs
+    
+    # Load Schema for Scalar DL install with a release name `load-schema`
+    $ helm upgrade --install load-schema scalar-labs/schema-loading --namespace default -f schema-loading-custom-values.yaml
+    
+    # Install Scalar DL with a release name `my-release-scalardl`
+    $ helm upgrade --install my-release-scalardl scalar-labs/scalardl --namespace default -f scalardl-custom-values.yaml
+    ```
+
+Note:
+
+* The same commands can be used to upgrade the pods.
+* Release name `my-release-scalardl` can be changed as per your convenience.
+* `helm ls -a` command can be used to list currently installed releases.
+* You should confirm the load-schema deployment has been completed with `kubectl get po -o wide` before installing Scalar DL.
+* Follow the [Maintain the cluster](./Maintaincluster.md) section for more customization.
+
+## Step 5. Monitor the Cluster
+
+It is critical to actively monitor the overall health and performance of a cluster running in production. 
+You can use Container Insights to collect performance metrics and Fluent Bit to collect logs of the EKS cluster.
+This section will help you to configure monitoring and logging for your EKS cluster.
+
+### Recommendations
+
+* Monitoring should be enabled for EKS in production.
+* CloudWatch agent should be configured in the EKS cluster for collecting metrics from pods.
+* Fluent Bit should be configured in the EKS cluster for collecting logs from pods.
+    * Node instance role must have `CreateLogGroup`, `CreateLogStream` and `PutLogEvents` permission for Fluent bit.
+
+### Set up Container Insights
+
+CloudWatch Container Insights helps you to collect, aggregate, and summarize metrics and logs from your containerized applications. Set up CloudWatch agent in the node group created for log and metrics collection.
+
+[Setup CloudWatch agent to collect cluster metrics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-setup-metrics.html) will help you to collect metrics from your containers.
+
+### Set up the Fluent Bit
+
+Fluent Bit allows you to collect logs from containerized applications and route logs to Amazon CloudWatch, you can set up the Fluent Bit in the node group created for log and metrics collection.
+
+[Setting up Fluent Bit](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-setup-logs-FluentBit.html#Container-Insights-FluentBit-setup)  will help you to collect logs from containers.
+
```

Comment:
The quick start guide installs the cloudwatch agent and fluent-bit all at once, so I think this is simpler and better!
https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-setup-EKS-quickstart.html

Reply:
Combined Container insights and Fluent Bit setup.